repose-aggregator/components/filters/uri-stripper-filter/src/main/scala/org/openrepose/filters/uristripper/UriStripperFilter.scala

```diff
@@ -65,9 +72,9 @@ class UriStripperFilter @Inject()(configurationService: ConfigurationService)
     } else {
       val wrappedRequest = new HttpServletRequestWrapper(servletRequest.asInstanceOf[HttpServletRequest])
       val wrappedResponse = new HttpServletResponseWrapper(
-        servletResponse.asInstanceOf[HttpServletResponse], ResponseMode.MUTABLE, ResponseMode.PASSTHROUGH)
+        servletResponse.asInstanceOf[HttpServletResponse], ResponseMode.MUTABLE, ResponseMode.MUTABLE)
```

Comment:
Yeah, I kind of figured since the headers mode was already mutable, we may as well just always set the body mode to mutable as well. I can make it "smarter" if you'd like, but the only way we could prevent this filter from breaking the stream is if we somehow knew before passing the response object along whether or not the response coming back contains a `Location` header.